[IMP] inventory: sendcloud shipping rules (#10112)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst

>    When return labels are created, Sendcloud will automatically charge the configured Sendcloud

Reviewer: ```suggestion
   When return labels are created, Sendcloud automatically charges the configured Sendcloud
```

   account.

Shipping rules
--------------

Optionally, create shipping rules to automatically generate shipping labels tailored to different

> products' needs. For example, a shipping rule can be created for customers shipping expensive

Reviewer: ```suggestion
product needs. For example, a shipping rule can be created for customers shipping expensive
```
just sounds a bit cleaner this way -- totally optional, though. completely your call. 👍

jewelry items to purchase insurance.

> To use shipping rules, navigate to :menuselection:`Inventory app --> Configuration --> Shipping

Reviewer: ```suggestion
To use shipping rules, navigate to :menuselection:`Inventory app --> Configuration --> Delivery: Shipping
```

> Methods` and select the intended `Sendcloud` shipping method.

Reviewer: ```suggestion
Methods`, and select the intended `Sendcloud` shipping method.
```

> In the :guilabel:`Sendcloud Configuration` tab, in the :guilabel:`Use Sendcloud shipping rules`
> field, choose the kind of shipments the shipping rules apply to: :guilabel:`Shipping` to customers,
> :guilabel:`Returns`, or :guilabel:`Both`.

Reviewer 2: ```suggestion
In the :guilabel:`OPTIONS` section of the :guilabel:`Sendcloud Configuration` tab,  in the :guilabel:`Use Sendcloud shipping rules`
field, choose the kind of shipments the shipping rules apply to: :guilabel:`Shipping` to customers,
:guilabel:`Returns` from cusotmers, or :guilabel:`Both`.
```

> In the :guilabel:`OPTIONS` section of the :guilabel:`Sendcloud Configuration` tab,  in the :guilabel:`Use Sendcloud shipping rules`
> field, choose the kind of shipments the shipping rules apply to: :guilabel:`Shipping` to customers,
> :guilabel:`Returns` from customers, or :guilabel:`Both`.

Reviewer: ```suggestion
Under the :guilabel:`Sendcloud Configuration` tab, in the :guilabel:`OPTIONS` section, choose the kind
of shipments the shipping rules apply to, via the :guilabel:`Use Sendcloud shipping rules` field.

From here, choose either: :guilabel:`Shipping` to customers, :guilabel:`Returns` from customers, or :guilabel:`Both`.
```

.. image:: sendcloud_shipping/enable-shipping-rules.png
   :align: center
   :alt: Use Shipping Rules field.

Then, in the Sendcloud website, navigate to :menuselection:`Settings --> Shipping rules`. Create a
new shipping rule by clicking :guilabel:`Create New`.

> Set a :guilabel:`Condition` to determine when the rule applies, in the :guilabel:`Actions` section,
> configure what to do when packages meet the condition.

Reviewer 2: ```suggestion
Set a :guilabel:`Condition` to determine when the rule applies, in the :guilabel:`Actions` section, then
configure what to do when packages meet the condition.
```

> Set a :guilabel:`Condition` to determine when the rule applies, in the :guilabel:`Actions` section, then
> configure what to do when packages meet the condition.

Reviewer: ```suggestion
In the :guilabel:`Actions` section, set a :guilabel:`Condition` to determine when the rule applies. Then,
configure what to do when packages meet the condition.
```
